Dockerfile

```
@@ -1,2 +1,9 @@
 FROM alpine:latest
+RUN <<EOF
+    apk update
+    apk upgrade \
+        libcrypto3 \
+        libssl3
+    apk cache clean
+EOF
```

That's unsafe because if one command fails the subsequent commands are still running and the instruction might succeed although one of the first commands actually failed. To make it safe, you could let the instruction fail if any command fails by specifying `set -e` at the beginning of the shell script, e.g.: `RUN set -ex; apk update; apk upgrade ...`
However, in this case there is a simpler way:
```
RUN apk upgrade --no-cache --update libcrypto3 libssl3
```

---

thanks,  I fixed the mistake